src/schemas/account.graphql

```diff
@@ -355,6 +366,14 @@ type UpdateAccountPayload {
   clientMutationId: String
 }
 
+type UpdateGroupsForAccountsPayload {
+  "The same string you sent with the mutation params, for matching mutation calls with their responses"
+  clientMutationId: String
+
+  "The accounts that were modified"
+  accounts: [Account]
```

Comment:
You can include `!` after this one because it will always return an array, even if it might be empty.